database/index.js

```diff
@@ -40,9 +40,8 @@ const makeEntry = (data) => {
   });
 };
 
-const search = (gameId, CB) => {
-  console.log('ID: ', gameId);
-  dlc.find({ associatedGameId: gameId }, (err, doc) => {
+const search = (param, CB) => {
```

Comment:
Why is "CB" capitalized? My guess is that's why its a different syntax highlight when its just a callback. This might create problems down the road.

Also, I couldn't add this comment at the top of this page for some reason, but will ever need to use lines 3 - 11 again? If not, then it's just dead weight and it could be removed.

Reply:
There wasnt a particular reason, i was just being lazy and have been setting my callbacks at CB, I do agree it should follow the same syntax as my regular code and will change it to `callback`.

As for lines 3-11, thats my bad. I had migrated those lines to `server/index.js` and forgot to delete the comments before I did the PR.